shared/chat/inbox/index.desktop.js

```diff
@@ -179,10 +180,14 @@ class Inbox extends React.PureComponent<Props, State> {
   }, 200)
 
   _scrollToUnread = () => {
-    if (this._firstOffscreenIdx <= 0 || !this._list) {
+    if (this._firstOffscreenIdx <= 0 || !this._scrollDiv.current) {
       return
     }
-    this._list.scrollToItem(this._firstOffscreenIdx, 'center')
+    let top = 100 // // give it some space below
```

Comment:
:sheep: double //.  And, I tried out top = 25 and I prefer it myself, but it's totally your call!

Reply:
Ah, thanks. 
How far it scrolls depends a bit on how big the item you started at is, so with smaller numbers occasionally it wouldn't scroll enough to fully show the unread row. I think 100 makes the variation less noticeable because it's ≫ our row heights.